packages/backend-tasks/src/tasks/PluginTaskSchedulerImpl.ts

```diff
@@ -48,6 +49,20 @@ export class PluginTaskSchedulerImpl implements PluginTaskScheduler {
     await TaskWorker.trigger(knex, id);
   }
 
+  private parseDuration(
+    frequency: TaskScheduleDefinition['frequency'],
+  ): string {
+    if ('cron' in frequency) {
+      return frequency.cron;
+    }
+
+    if (Duration.isDuration(frequency)) {
+      return frequency.toISO();
+    }
+
+    return Duration.fromObject(frequency).toISO();
```

Comment:
no tests actually exercise this new code

Reply:
moved this out to a separate function to test it. The tests in TaskWorker.test.ts was deeper down and used Duration directly. The existing Impl test did not exercise this in its current form.

Unrelated but should this be renamed to DefaultTaskScheduler? 🤔

Reply:
There will only ever be one implementation though, and it's internal.